utils/src/main/java/org/jboss/windup/util/Iterables.java

```diff
@@ -0,0 +1,17 @@
+package org.jboss.windup.util;
+
+/**
+ * A windup specific class with some useful methods for iterables
+ */
+public class Iterables
```

Comment:
Why reproduce this here? I believe that we already use guava Iterables.size() in a few places.